Civi/Events/ParticipantSubscriber.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/*-------------------------------------------------------+
+| BUND Event Customisations                              |
+| Copyright (C) 2023 SYSTOPIA                            |
+| Author: C. Jana (jana@systopia.de)                     |
+| http://www.systopia.de/                                |
++--------------------------------------------------------+
+| This program is released as free software under the    |
+| Affero GPL license. You can redistribute it and/or     |
+| modify it under the terms of this license which you    |
+| can read by viewing the included agpl.txt or online    |
+| at www.gnu.org/licenses/agpl.html. Removal of this     |
+| copyright header is strictly prohibited without        |
+| written permission from the original author(s).        |
++--------------------------------------------------------*/
+
+namespace Civi\Events;
+
+use Civi\Api4\Contact;
+use Civi\Api4\Email;
+use Civi\Api4\Participant;
+use Civi\Api4\Event;
+use Civi\Core\Service\AutoSubscriber;
+use Civi\Core\Event\GenericHookEvent;
+use CRM_Core_BAO_MessageTemplate;
+
+// TODO: Since when does the class AutoSubscriber exist? Modify minimum CiviCRM Version accordingly
+
+class ParticipantSubscriber extends AutoSubscriber {
+  /**
+   * The id of the contact that receives notifications via email.
+   * This should be a person working at BUND. Currently: Felix Schwalbe.
+   */
+  private int $recipient_bund_id = 2886;
```

Comment:
Since that's a constant value, this could be made a class constant. However, I'd prefer either making it configurable or reading it from a setting (with the constant as a default) so that it's easier to override.